lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -490,6 +491,8 @@ class IdentityCreatorController extends BaseController {
     if (context.mounted) {
       if (filePickerResult?.files.isNotEmpty == true) {
         final platformFile = filePickerResult!.files.first;
+        // __file_picker_web-file-input
+        html.querySelector('#__file_picker_web-file-input')?.children.clear();
```

Comment:
please remove it

Reply:
Why?

Reply:
> Why?

Who are you asking? What are you asking about?

Reply:
use for testing. Removed